stories/otp-input/doc-elements.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+    display: flex;
+    flex-direction: column;
+    max-width: 500px;
```

Comment:
use `rem`

Reply:
temp stories for standalone, might no longer need this if we move the stories to form

Reply:
I dont foresee it will be moved to `Form`. In that case perhaps using `<StoryContainer>` will help?